deploy/manifests/kip/overlays/mac/anka_svc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -o registry-manifests
+    kompose.version: 1.22.0 (955b78124)
+  labels:
+    io.kompose.service: anka-registry
```

Comment:
What are all the "kompose" labels/annotations?

Reply:
I generated those manifests from docker-compose using [kompose](https://kompose.io/). Changed to recommended k8s labels